deploy/vagrant/create

```diff
@@ -7,14 +7,14 @@ usage() {
  exit 1
 }
 
-if [[ "$#" != "2" ]]; then
+if [ "$#" -ne 2 ]; then
  usage
 fi
 
 TOTAL="$1"
 PROVIDER="$2"
 
-if [[ "$TOTAL" < "2" ]]; then
+if [ "$TOTAL" -lt 2 ]; then
```

Comment:
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create 17 aws
ERROR: The environment variable AWS_ACCESS_KEY_ID must be set
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create 1 aws
number of machines should at least be 2
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create -1 aws
number of machines should at least be 2
oldnine@osboxes:~/git/tachyon/deploy/vagrant$ ./create -99 aws
number of machines should at least be 2